src/EcTranslators/EcTranslators.csproj

```diff
@@ -0,0 +1,133 @@
+    <Optimize>true</Optimize>
+    <DebugType>pdbonly</DebugType>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <LangVersion>7.3</LangVersion>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualBasic" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="BingTranslatorAutoConfigDialog.cs">
+      <SubType>Form</SubType>
+    </Compile>
+    <Compile Include="BingTranslatorAutoConfigDialog.designer.cs">
+      <DependentUpon>BingTranslatorAutoConfigDialog.cs</DependentUpon>
+    </Compile>
+    <Compile Include="BingTranslatorEncConverter.cs" />
+    <Compile Include="BingTranslatorEncConverterConfig.cs" />
+    <Compile Include="BingTranslatorSchemas.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+    <Compile Include="Properties\Settings.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTimeSharedInput>True</DesignTimeSharedInput>
+      <DependentUpon>Settings.settings</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ECInterfaces\ECInterfaces 2010.csproj">
+      <Project>{2935e02f-43ed-4670-9a00-a91b0ad3d66c}</Project>
+      <Name>ECInterfaces 2010</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\EncCnvtrs\EncCnvtrs 2010.csproj">
+      <Project>{3e4b908a-4f96-43c5-981b-1aa4b73ccc26}</Project>
+      <Name>EncCnvtrs 2010</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="BingTranslatorAutoConfigDialog.resx" />
+    <EmbeddedResource Include="Properties\Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\..\redist\EC\Plugins\Translators 4.0.0.0 Plugin Details.xml">
+      <Link>PlugIn\Translators 4.0.0.0 Plugin Details.xml</Link>
+    </None>
+    <Content Include="TRFFC17.ICO" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="app.config" />
+    <None Include="FieldWorks.snk" />
+    <None Include="Properties\Settings.settings">
+      <Generator>SettingsSingleFileGenerator</Generator>
+      <LastGenOutput>Settings.Designer.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json">
```

Comment:
This kind of reference is meant for new style .csproj files, but I suppose it works this way for windows builds using VS2019.
If you added Newtonsoft.Json to the EncCnvtrs\packages.config file and referenced the assembly with a normal package reference it should fix your linux build.

Reply:
my earlier attempts (if you look higher in the pull request commit list) had it being pulled by nuget in the packages.config file, first v13 and when that didn't work, v12, then adding a redirect for it to v12 in the app.config... none of those permutations worked. So all the latter ones you discuss here are "reaching at straws" trying to make it work. I can revert it back to that scenario (in packages.config) and someone can try to fix it from there?